ci-operator/config/redhat-developer/odo/redhat-developer-odo-main.yaml

```diff
@@ -8,18 +8,27 @@ resources:
       cpu: "2"
       memory: 4Gi
 tests:
-- as: unit
+- as: unit-and-validate-test
   skip_if_only_changed: ^website/|docs/|\.md$|\.adoc$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
   steps:
+    pre:
+    - as: install-golint
+      commands: make goget-tools
+      from: src
+      resources:
+        requests:
+          cpu: "2"
+          memory: 4Gi
+      timeout: 4h0m0s
```

Comment:
Are we setting 4 hours' timeout for installing golint?

Reply:
Thanks, @dharmit, that was a copy and paste mistake, I updated the main test timeout but forgot to update this one. Updating it ASAP